mkosi

```diff
@@ -2830,25 +2865,46 @@ def check_root():
     if os.getuid() != 0:
         die("Must be invoked as root.")
 
+def run_shell(args):
+    target = "--directory=" + args.output if args.output_format in (OutputFormat.directory, OutputFormat.subvolume) else "--image=" + args.output
+
+    cmdline = ["systemd-nspawn",
+               target]
+
+    if args.verb == "boot":
+        cmdline += ('--boot',)
```

Comment:
It would be nice to have an option to run another command instead of `/bin/sh` (which is the very limited dash in debian-based systems)